variables.md

```diff
@@ -126,6 +126,7 @@ SYSAUTHTEST | boolean | false | Enable system authentication test (`sysauth/sssd
 TEST | string | | Name of the test suite.
 TOGGLEHOME | boolean | false | Changes the state of partitioning to have or not to have separate home partition in the proposal.
 TUNNELED | boolean | false | Enables the use of normal consoles like "root-consoles" on a remote SUT while configuring the tunnel in a local "tunnel-console"
+TYPE_BOOT_PARAMS_FAST | boolean | false | Allows test developer to speed up local verification
```

Comment:
How is max_interval value a boolean ?

Reply:
As you are using `get_var` which is getting value of the variable, I would use the (max_interval) value when defined otherwise VERY_SLOW_TYPING_SPEED
`type_string(" @params ", max_interval => get_var('TYPE_BOOT_PARAMS_FAST', utils::VERY_SLOW_TYPING_SPEED));`

Reply:
I also recommend to use a different variable name. The name still sounds like a boolean when it isn't anymore. How about just replacing the existing constant "SLOW_TYPING_SPEED" with a test variable of the same name that you just set to a default value of 13 but can override it?

Reply:
It is a boolean.

You want to type fast? -> TYPE_BOOT_PARAMS_FAST=1
You want default (slow)? -> no setting or TYPE_BOOT_PARAMS_FAST=0